ci/azure-pipelines.yml

```diff
@@ -153,8 +159,18 @@ stages:
             workingDirectory: $(System.DefaultWorkingDirectory)/ci/vm/
 
           - bash: |
+              set -x
+              hostname
+              export VAGRANT_HOME=/opt/vagrant_home
               vagrant destroy -f
               vagrant global-status --prune
+              set +e
```

Comment:
Why do we need this?

Reply:
Because of this: https://github.com/openucx/xpmem/pull/38#issuecomment-1667610858
These commands are for "killing confirmation" and have a high risk of failure if resources were cleaned up already.